src/containers/CourseOverview/CourseOverview.jsx

```diff
@@ -1,16 +1,14 @@
-import React, {useState} from "react";
-import CourseOverviewList from "../CourseOverviewList/CourseOverviewList";
-import VideoCardList from "../VideoCardList/VideoCardList";
-import "./CourseOverview.scss";
-import MenuBar from "../../components/MenuBar/MenuBar";
-import BackButton from "../../components/BackButton/BackButton";
-import Navigation from "../../components/Navigation/Navigation";
-import dummyVideoData from "../../assets/data/dummyVideoCardData";
+import React, { useState } from 'react';
+import CourseOverviewList from '../CourseOverviewList/CourseOverviewList';
+import './CourseOverview.scss';
+import MenuBar from '../../components/MenuBar/MenuBar';
+import BackButton from '../../components/BackButton/BackButton';
+import Navigation from '../../components/Navigation/Navigation';
 
 // path: /courses/:courseId
```

Comment:
Commented out code?

Reply:
It's just being left in so people have easy access to the path of the page